### PR TITLE
chore(Core): Alphabetically organize #includes

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  Alphabetically organize #includes

## Tests Performed:
- Build

## How to Test the Changes:

CI Build

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: src/server/game/AI/CreatureAIRegistry.cpp [hunk 1]
```diff
@@ -4,18 +4,18 @@
...
-#include "MovementGeneratorImpl.h"
-#include "CreatureAIRegistry.h"
-#include "WaypointMovementGenerator.h"
-#include "CreatureAIFactory.h"
+#include "ReactorAI.h"
 #include "SmartAI.h"
+#include "TotemAI.h"
+#include "WaypointMovementGenerator.h"
 
 //#include "CreatureAIImpl.h"
```
Comment (Winfidonarleyan) [inline src/server/game/AI/CreatureAIRegistry.cpp:20]: to delete

File: src/server/game/AI/SmartScripts/SmartScript.h [hunk 2]
```diff
@@ -10,12 +10,11 @@
 #include "Common.h"
 #include "Creature.h"
 #include "CreatureAI.h"
-#include "Unit.h"
-#include "Spell.h"
 #include "GridNotifiers.h"
-
-#include "SmartScriptMgr.h"
 //#include "SmartAI.h"
```
Comment (Winfidonarleyan) [inline src/server/game/AI/SmartScripts/SmartScript.h:14]: to delete

File: src/server/game/Warden/WardenMac.cpp [hunk 3]
```diff
@@ -4,19 +4,19 @@
  * Copyright (C) 2005-2009 MaNGOS <http://getmangos.com/>
  */
 
-#include "Cryptography/WardenKeyGeneration.h"
+#include "ByteBuffer.h"
 #include "Common.h"
-#include "WorldPacket.h"
-#include "WorldSession.h"
+#include "Cryptography/WardenKeyGeneration.h"
```
Comment (Winfidonarleyan) [inline src/server/game/Warden/WardenMac.cpp:9]:
```cpp
#include "WardenKeyGeneration.h"
````

File: src/server/game/Warden/WardenMac.h [hunk 4]
```diff
@@ -7,11 +7,11 @@
 #ifndef _WARDEN_MAC_H
 #define _WARDEN_MAC_H
 
+#include "ByteBuffer.h"
 #include "Cryptography/ARC4.h"
```
Comment (Winfidonarleyan) [inline src/server/game/Warden/WardenMac.h:11]:
```cpp
#include "Cryptography/ARC4.h"
```

File: src/server/game/Warden/WardenMac.h [hunk 5]
```diff
@@ -7,11 +7,11 @@
...
-#include <map>
 #include "Cryptography/BigNumber.h"
```
Comment (Winfidonarleyan) [inline src/server/game/Warden/WardenMac.h:12]:
```cpp
#include "BigNumber.h"
```

**Comment by Winfidonarleyan** [comment]
Very good

**Comment by Winfidonarleyan** [comment]
Did you do it manually? I hope not.

**Comment by Kitzunu** [comment]
Well, kind of.
I found a plugin on VSCode that orders it with shft+ctrl+a, but I need to go through every file

File: src/server/game/Warden/WardenMac.h (hunk 4 above)
Comment (Kitzunu) [reply src/server/game/Warden/WardenMac.h:11]: It's already there o.o

**Comment by Kitzunu** [comment]
I have no idea why it fails on a file I never touched q.q
```cpp
uint32 const EncounterCount         = 23;
``` 
already exist in the core, so why fail now D:

**Comment by Winfidonarleyan** [comment]
Hmm, very interest

**Comment by locus313** [comment]
@Kitzunu the failures should be resolved now.

**Comment by Kitzunu** [comment]
I have no idea how you figured it out, but thanks 😍

**Review by FrancescoBorzi — APPROVED** [review]

**Review by Winfidonarleyan — APPROVED** [review]

**Pull request merged**